include/nds/arm9/video.h

```diff
@@ -54,8 +54,8 @@
 	tile and map offsets will be lost.
 
 	VRAM banks can be mapped to specific addresses for specific purposes.  In our
-	case, any of the 4 main banks amd several of the smaller ones can be mapped to
-	the main 2D background engine.(A B C and D banks are refered to as ìmainî
```

Comment:
If I had to guess, they probably got copied from the left/right single/double quotes ‘’/“” from a different character encoding.